internal/common/filespec.go

```diff
@@ -219,7 +224,24 @@ func (this *FileSpec) CopyTarget(baseDir string, destDir string, onlyContents bo
 		// no-op.
 		return nil
 	case FILESPEC_TYPE_PATH:
-		return this.copyPath(baseDir, destDir, onlyContents)
+		if !strings.ContainsAny(this.Path, `[]\*^?`) {
```

Comment:
I still don't like this.
You try to separate glob and non-glob paths, but then you include non-glob paths in the glob case, e.g., `/some/file/with_a_\*_star.txt`.
Making it seem like you separate the code into two cases when you don't is a recipe for trouble and hard to debug issues.